dopamine/discrete_domains/run_experiment.py

```diff
@@ -625,9 +626,18 @@ def _run_one_iteration(self, iteration):
       iteration: int, current iteration number, used as a global_step for saving
         Tensorboard summaries
     """
-    statistics = super(AsyncRunner, self)._run_one_iteration(iteration)
+    statistics = iteration_statistics.IterationStatistics()
+    tf.logging.info('Starting iteration %d', iteration)
+    num_episodes_train, average_reward_train = self._run_train_phase(
+        statistics)
+    num_episodes_eval, average_reward_eval = self._run_eval_phase(
+        statistics)
     with self._output_lock:
-      self._log_experiment(iteration, statistics)
-      self._checkpoint_experiment(iteration)
+      self._log_experiment(self._completed_iteration, statistics)
```

Comment:
This could use a short comment talking citing the reordering and talking about its intention.

Reply:
Reverted the reordering. Here the change is just  about replacing `iteration` with `self._completed_iteration`.

Reply:
I don't remember how it looked before, but this is really straightforward.

Reply:
Bleh -- sorry.  I'd misused language earlier.  I'd meant to ask for a comment explaining the why of `completed_iteration`.

Reply:
Done. Added a comment.

Reply:
That comment is great.